isis/src/base/objs/SpiceRotation/SpiceRotation.cpp

```diff
@@ -399,6 +400,24 @@ namespace Isis {
    */
   void SpiceRotation::LoadCache(double time) {
     LoadCache(time, time, 1);
+  }
+
+
+  void SpiceRotation::LoadCache(json &isd){
+    // Clear any existing cached data to make it reentrant (KJB 2011-07-20).
+    p_timeFrames.clear();
+    p_TC.clear();
+    p_cache.clear();
+    p_cacheTime.clear();
+    p_cacheAv.clear();
+    p_hasAngularVelocity = false;
+ 
```

Comment:
I'm guessing this is just partially implemented, just want to make sure.

Reply:
Yes, should be removed for now.